### ISSUE TITLE
[Bug] TextInput component updates

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#1086]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

NA

### OS Version

NA

### App version

0.21.0

### App build number

1761081723

### App build type

beta

### Describe the issue

1. when there is no placeholder, and the label is long it shows the entire content and the field resizes
2. when there is a placeholder, and the label is long it shows the entire content and the field resizes

### Expected Behavior

1. when there is no placeholder, the Label should only show 2 lines and then there should be a truncation - the field should stay of the same size
2.  when there is a placeholder, the Label should only show 1 line and then there should be a truncation - the field should stay of the same size

### Accessibility settings

NA

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1762898813**
- Build tag: **9176741**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1763676438**
- Build tag: **4247640**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1764022049**
- Build tag: **7794df8**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
